### DISCUSSION TITLE
Image updater does not reconcile a deployment on EKS

### DISCUSSION THREAD

**Discussion by billal-ayyoob** [argoproj/argo-cd#21141] [Q&A]
My application resource contains the required annotations for image updater to perform rolling updates or reconciles. I have followed the git write-back method and created a parameter store file .argocd-source-<appName>.yaml as well.

Inside this app resource there are two sources one for dagster helm chart and the other one for the path of the customized values file for it on my Gitops repository. With the sane dagster helm chart it deploys one other deployment for user code in which I ran my data pipelines as sub-chart. It retrieves and runs the docker images from my ECR repository. 

Issue:
Whenever a new image is pushed to the ECR with the updated code for Data pipeline that deployed through the dagster chart should be updated by image updater. But it neither throws any error nor performs updates.

Expected behaviour:

Rolling update the image version for deployed sub-chart

Can argocd image updater perform reconciliation for the sub-chart that has been deployed through a single application resource?

Am I missing some configs?

Thank you.

**Comment by chengfang** [comment]
@billal-ayyoob can you move this discussion to https://github.com/argoproj-labs/argocd-image-updater/issues, to get more input from image-updater community?  Also see same discussion in [image-updater slack channel](https://cloud-native.slack.com/archives/C0296T47CHY/p1733994047100909)

**Reply by billal-ayyoob** [reply]
Yes, up to discussing the same in slack channel for image updater.
Thanks
